src/components/Login/SignUp.js

```diff
@@ -109,9 +110,32 @@ class SignUp extends Component {
 
   onLoginCallback(result, error) {
     if (error) {
-      this.alert('Sever error(Profile)! Please sign in again.');
+      this.alert('Please check your account information and sign in again!');
```

Comment:
This logic is same with a login in Login.js
I just moved this logic into SignUp.js

This error is about Facebook sign-in.
But, this error is occurred after calling signing api.
You can refer this on line 30 in UserUtil.js.